tests/baselines/reference/destructureOptionalParameter.js

```diff
@@ -36,11 +36,11 @@ function f2(_a) {
 declare function f1({a, b}?: {
     a: number;
     b: string;
-}): void;
+} | undefined): void;

```

Comment:
Seems sort of redundant to include `undefined` when we already have a `?` on the parameter. Same in a number of cases below.